qtconsole/console_widget.py

```diff
@@ -207,8 +207,7 @@ def __init__(self, parent=None, **kw):
         parent : QWidget, optional [default None]
             The parent for this widget.
         """
-        QtGui.QWidget.__init__(self, parent)
-        LoggingConfigurable.__init__(self, **kw)
+        super(ConsoleWidget, self).__init__(**kw)
```

Comment:
Thanks. Can you tell me what problem that would cause? I've been using this for a while, and I can't see any misbehavior.